solutions/client-setup/namespaces/client-name-hierarchy.yaml

```diff
@@ -117,3 +117,21 @@ subjects:
   - name: cnrm-controller-manager-client-name-hierarchy # kpt-set: cnrm-controller-manager-${client-name}-hierarchy
     namespace: cnrm-system
     kind: ServiceAccount
+---
+# Grant viewer role on the client-name-hierarchy namespace to policies K8S SA
```

Comment:
are we missing a client-name-policies namespace ?

Reply:
not sure if it's missing or if it doesn't exist for a reason?
either way it's not there

Reply:
internal bug opened 5804